caput/scripts/runner.py

```diff
@@ -190,8 +194,11 @@ def template_run(templatefile, var):
             tfh.write(expanded_string)
             tfh.flush()
 
-            # Run the pipeline
-            Manager.from_yaml_file(tfh.name).run()
+            # Run or queue the pipeline
+            if queue:
```

Comment:
Where does `queue` come from?

Reply:
Haha. Good spot. I already fixed that one but I haven't pushed the commit. I found it when it submitted a bunch of jobs into the queue that I had wanted it to run immediately!

Reply:
Ah! Okay, once that is pushed, we can merge this in. =)